app/controllers/api/users_controller.rb

```diff
@@ -0,0 +1,13 @@
+module Api
+  class UsersController < Api::BaseController
+    def watching
+      submission_id = params[:submission_id]
+      current_user.update_attributes(
+        last_watched_submission_id: submission_id,
+        last_watched_submission_at: DateTime.now
+      )
+
+      render json: User.current_watching_for(submission_id, current_user.id).to_a.map(&:nickname)
```

Comment:
@lesniakania I think it's good to use `pluck` here http://api.rubyonrails.org/classes/ActiveRecord/Calculations.html#method-i-pluck